client/landing/stepper/declarative-flow/helpers/test/index.ts

```diff
@@ -0,0 +1,29 @@
+import { shouldRedirectToSiteMigration } from '..';
+import { STEPS } from '../../internals/steps';
+describe( 'DeclarativeFlowHelpers', () => {
+	it( 'returns true when current step, platform and origin as set properly', () => {
+		expect(
+			shouldRedirectToSiteMigration(
+				STEPS.IMPORT_LIST.slug,
+				'wordpress',
+				'site-migration-identify'
+			)
+		).toBe( true );
+	} );
+
+	it( 'returns false when current step is not importLimit', () => {
+		expect(
+			shouldRedirectToSiteMigration( 'other-step', 'wordpress', 'site-migration-identify' )
+		).toBe( false );
+	} );
+
+	it( 'returns false when platform is not wordpress', () => {
+		expect(
+			shouldRedirectToSiteMigration(
+				STEPS.IMPORT_LIST.slug,
+				'other-platform',
+				'site-migration-identify'
```

Comment:
Should we add a test for other `origin` values as well? I feel that testing `''` or `null` would be important as those should be relatively common cases.

Reply:
Also, should we not use the slug for the origin as well?
```suggestion
				STEPS.SITE_MIGRATION_IDENTIFY.slug
```

Reply:
I added more test cases here d1ed86a757e1de9a9bf4840b3b38b35d58e5e4ed